modules/service/src/main/scala/lucuma/itc/input/odb/RadialVelocityInput.scala

```diff
@@ -26,17 +26,22 @@ object RadialVelocityInput {
         val rMetersPerSecondʹ      =
           OptionT(rMetersPerSecond).semiflatMap(resultFromMetersPerSecond).value
         val rKilometersPerSecondʹ  =
-          OptionT(rKilometersPerSecond).semiflatMap(resultFromKilometersPerSecond).value
-        (rCentimetersPerSecondʹ, rMetersPerSecondʹ, rKilometersPerSecondʹ).parTupled.flatMap {
-          case (centimetersPerSecond, metersPerSecond, kilometersPerSecond) =>
+          OptionT(rKilometersPerSecond)
+            .semiflatMap(resultFromKilometersPerSecond)
+            .value
+
+        // comment to prevent code formatter from merging with the previous
+        // definition
+        (rCentimetersPerSecondʹ, rMetersPerSecondʹ, rKilometersPerSecondʹ).parTupled
+          .flatMap { case (centimetersPerSecond, metersPerSecond, kilometersPerSecond) =>
```

Comment:
Scalafmt really wants to turn this into one long definition of `rKilometersPerSecond'` by merging `.value` with the `(rCentimetersPerSecondʹ, rMetersPerSecondʹ, rKilometersPerSecondʹ)`.  Adding an intervening comment seems to prevent it.  :-/

Reply:
![](https://i0.wp.com/adailypinch.com/wp-content/uploads/Screen-Shot-2015-01-24-at-11.30.28-PM.png)

Reply:
You can tell scalafmt to omit a chunk of code by enclosing it in commented directives:
```scala
// format: off
...
// format: on
```